### PR TITLE
Deleted unused failRegexEvent

### PR DESCRIPTION
**Pull request by liliyao2022**
Deleted a  variable because it is declared but its value is never read.

### CODE REVIEW

File: jsx/src/components/AddUser/AddUser.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const AddUser = (props) => {
     });
   };
 
-  var { addUsers, failRegexEvent, updateUsers, history } = props;
```
Comment (minrk) [inline jsx/src/components/AddUser/AddUser.jsx:24]: I think this is unused everywhere - can you remove every reference to it, not just this file?

Comment (liliyao2022) [reply jsx/src/components/AddUser/AddUser.jsx:24]: OK, no problem. I'll try to find all of them.

Comment (liliyao2022) [reply jsx/src/components/AddUser/AddUser.jsx:24]: Hi, @minrk Please kindly review my other PRs, thank you!

**Comment by minrk** [comment]
Thank you for suggesting changes across several files, but please make one change in one PR (this PR should remove all failRegexEvent), rather than opening one or more PRs per file with the same changes.

You now have [5 open PRs](https://github.com/jupyterhub/jupyterhub/pulls/liliyao2022) that have been reviewed and are awaiting changes from you. Please address the review in those PRs before opening any new ones.

**Comment by liliyao2022** [comment]
Thank you for your suggestions @minrk, you are definitely right and I should have made all these changes in just one PR, however, I was a little bit unfamiliar with how to do this, sorry for any inconvenience. Could you tell me how to make the PR as you advised?  Should I use the git command to commit these changes?

**Comment by minrk** [comment]
> Should I use the git command to commit these changes?

Yes, you can take all the changes from the PRs you've made, and add them via git. You can find more [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

**Comment by liliyao2022** [comment]
Hey @minrk, after working on this PR and doing my research, I've learned a lot about Git and how to make contributions to an Open source project. Thank you so much! You are a really great mentor that I can trust!

**Pull request merged**